test/cctest/node_test_fixture.h

```diff
@@ -100,13 +85,15 @@ class NodeTestFixture : public ::testing::Test {
       uv_run(&current_loop, UV_RUN_ONCE);
     }
     v8::V8::ShutdownPlatform();
+    delete allocator_;
     delete platform_;
     platform_ = nullptr;
     CHECK_EQ(0, uv_loop_close(&current_loop));
   }
 
  private:
   node::NodePlatform* platform_ = nullptr;
+  v8::ArrayBuffer::Allocator* allocator_ = nullptr;
```

Comment:
How about using `unique_ptr` instead?

Reply:
Yeah, I should probably update all pointers in that file to use smart pointers. I'll update them. Thanks